tao_compiler/mlir/disc/transforms/lower_tf.cc

```diff
@@ -1080,6 +1083,161 @@ LogicalResult convertQuantizedConv2DWithBiasAndRequantizeOp(func::FuncOp func) {
   return success();
 }
 
+// Returns 1D 64-bit dense elements attribute with the given values.
+static mlir::DenseIntElementsAttr GetI64ElementsAttr(
```

Comment:
nit: do we have a common place for this kind of utility.

Reply:
I think maybe in some utils files is better, but I don't know if we already have one

Reply:
disc_util.h|cc?

Reply:
done